src/EditorFeatures/Core.Cocoa/InlineRename/CommandHandlers/RenameCommandHandler_RenameHandler.cs

```diff
@@ -35,7 +39,7 @@ public VSCommanding.CommandState GetCommandState(RenameCommandArgs args)
                 return VSCommanding.CommandState.Unspecified;
             }
 
-            var documents = textContainer.GetRelatedDocuments();
+            _ = textContainer.GetRelatedDocuments();
```

Comment:
💭 A follow-up issue to remove the items getting assigned to discards would be good.

Reply:
Definitely. Already filed https://github.com/dotnet/roslyn/issues/48871 to review all suppressions. I'll add a note to revisit discards too.